packages/keystone/README.md

```diff
@@ -75,12 +73,31 @@ const keystone = new Keystone({
 
 Note that `maxTotalResults` applies to the total results of all relationship queries separately, even if some are nested inside others.
 
-### `secureCookies`
+### `cookie`
 
-A secure cookie is only sent to the server with an encrypted request over the HTTPS protocol. If `secureCookies` is set to true (as is the default with a **production** build) for a Keystone project running on a non-HTTPS server (such as localhost), you will **not** be able to log in. In that case, be sure you set `secureCookies` to false. This does not affect development builds since this value is already false.
+_**Default:**_ see Usage.
+
+A description of the cookie properties is included in the [express-session documentation](https://github.com/expressjs/session#cookie).
+
+#### `secure`
+
+A secure cookie is only sent to the server with an encrypted request over the HTTPS protocol. If `secure` is set to true (as is the default with a **production** build) for a KeystoneJS project running on a non-HTTPS server (such as localhost), you will **not** be able to log in. In that case, be sure you set `secure` to false. This does not affect development builds since this value is already false.
 
 You can read more about secure cookies on the [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
 
+#### Usage
+
+```javascript
+const keystone = new Keystone({
+  /* ...config */
+  cookie: {
+    secure = process.env.NODE_ENV === 'production', // Default to true in production
+    maxAge = 1000 * 60 * 60 * 24 * 30, // 30 days
+    sameSite = false,
```

Comment:
I think you accidentally swapped the code example and the `keystone` ctor destructuring default code 🤔

Reply:
Oh, good catch 🤦